utils/validators.ts

```diff
@@ -273,6 +273,8 @@ export function parseTriggerPostData ({ userId, postData, postDataParameters }:
       .replace('<opReturn>', opReturn)
       .replace('<signature>', `${JSON.stringify(signature, undefined, 2)}`)
       .replace('<inputAddresses>', `${JSON.stringify(postDataParameters.inputAddresses, undefined, 2)}`)
+      .replace('<value>', `${JSON.stringify(postDataParameters.value)}`)
```

Comment:
Why are you JSON.stringifing a string? Just do as it's done with the other strings, e.g.
\`"${postDataParameters.address}"\`